data/projects/editor_support.md

```diff
@@ -1,31 +1,67 @@
-Develop an efficient strategy that allows coala to maintain a number of plugins
-for different editors.
+coala features quite a few editor plugins already, none of which are 
+*supported* by the definition of having regular continuous integration and 
+tests running to ensure its functionality. The goal of this project is to 
+plan, introduce and configure a testing routine that allows coala to ensure
+the functioning of it's editor plugins at all times.
+A viable approach might be to set up weekly tests for the current editor
+plugins.
 
-We consider any bit of code *supported* only if we have regular continuous
-integration and tests running to ensure it's functionality.
+Details (for testing): 
 
-coala features quite some editor plugins already, none of which are *supported*
-according to the above definition. The goal of this project is to plan,
-introduce and configure a testing routine that allows coala to ensure the
-functioning of it's editor plugins at all times.
-A viable approach might be to set up weekly tests for the current editor
-plugins. For some editors like sublime and gedit, running tests is not easy and
-there is no api to do it. In such cases we  would want to use dogtail
+For sublime, we'll be using the 
+[unittest plugin](https://github.com/randy3k/UnitTesting)
+which is a plugin by itself. 
+For atom, we have the apm testing architecture
+which does not require a GUI.
+For VI and emacs, we will most likely use a testing framework such as 
+[Vimcasts](https://github.com/kana/vim-vspec).
+For some editors like gedit, running tests is not easy and
+there is no api to do it. In such cases we  would want to use 
+[dogtail](https://fedorahosted.org/dogtail/)
 (a GUI tester) or a similar tool to do this.
 
+Details (for maintenance):
+
+We will have to explore the possibility of having a single git repository for 
+atom, sublime-text and vim as it will ultimately be easier to maintain in the
+long run and will have a proper CI, which can be easily managed. Issues will 
+be raised more frequently as a result of this.
+
+Details (for documentation):
+
+Documentation which isn't available at the moment will have to describe the 
+usage details of existing plugins, we can have asciicinemas or gifs of the
+entire process from installation to examples of the plugins in action.
+Documentation can also be created for future plugins, and on the basis of the
+tests which will be written for the existing ones, it'll be easier to test
+the future plugins. This will result in a clear cut path of creating plugins
+for coala.
+
+The goal is to develop an efficient strategy that allows coala to maintain a 
```

Comment:
maybe move this paragraph above the details paragraphs. It's kinda missing an introduction here after the details.